### DISCUSSION TITLE
QGIS Grids - Draw Coordinates Bug

### DISCUSSION THREAD

**Discussion by travisz09** [ua-gist604b-f24/syllabus#5] [General]
Hello all,

I am working on the intro to QGIS module 2-0. In tutorial 1, steps 21-26 we are adding some graticule lines to a map of Japan. I followed the instructions, but for some reason the coordinate labels are not showing up on the right side of my layout. In the screenshot below you can see that the labels are visible on the top, left, and bottom, but are missing on the right side. In the Grid Properties box to the far right, you can clearly see that the option for the "Right" side is set to "Show All", "Inside Frame". 

I tried restarting QGIS. I also tried fiddling with any related settings that I could think of, but nothing has worked. I have not been able to find anything about this bug online (most of the issues with grids in the forums seem to be related to crs). I am just going to ignore it and move forward for now, as it is not mission-critical, but I would be interested to hear if anyone else encounters this problem, or have any ideas to fix it.

Let me know what you think.

![image](https://github.com/user-attachments/assets/559c6955-d947-493b-b6ad-c6c12c4f7656)

Thanks all!

**Comment by travisz09** [comment]
I just realized this bug is also addressed in the comments of the tutorial. It looks like restarting might help, but it did not work for me.

I just left a comment there as well.

**Reply by jenw-ua** [reply]
I had the exact same problem with no solution, after trying all of the same things. Seems like a weird quirk.

**Reply by kev-bone** [reply]
Knowing that you both had the issue, I added the grid before I created the inset. That worked, for whatever reason.
![image](https://github.com/user-attachments/assets/746f54ef-27c3-47a6-a114-e5ea43db2967)

**Reply by aaryno** [reply]
weird bug. I wonder if it's just in the LTR version. Another thing to try is to resize the different frames and see if there is something obscuring them. It looks like your map panel is at least to the edge of the Map Layout. Another thing to do is to permute the various options and see if it starts working after changing it.
